components/Anchor.tsx

```diff
@@ -0,0 +1,35 @@
+import { HTMLAttributes, PureComponent } from 'react';
+import Link from 'next/link';
+
+import styles from '../styles/Anchor.module.scss';
+
+type AnchorProps = {
```

Comment:
Can we use an interface to declare prop types, then in the prop types section have the & HTMLAttributes<HTMLAnchorElement> (line 18)